cli/cmd/testdata/install_controlplane_tracing_output.golden

```diff
@@ -1875,7 +1875,6 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/config: 31ca92e63a48870d9b7cad9d7a613e2164b4b016577720744b2f0a1da3a3c844
```

Comment:
I remember that this is used to update the deployment whenever things changed in the `-rbac` file. Is it not needed with the CLI? :thinking:

Reply:
It would work for the full CLI install, but not for `linkerd install control-plane` nor when installing the `linkerd-control-plane` chart because the rbac file is in the other chart.